docs/external-id/customers/faq-customers.md

```diff
@@ -34,7 +34,7 @@ No, External ID add-ons don't have a free tier. However, the ID Governance premi
 
 ### Does External ID have phone authentication via SMS?
 
-Yes, this feature is now available in workforce and [external](concept-multifactor-authentication-customers.md) configurations.
+No, as per this [article](concept-multifactor-authentication-customers.md), "Currently, SMS isn't available for first-factor authentication or self-service password reset in external tenants".
```

Comment:
@rbrayb thanks for catching - we should clarify that SMS is now supported for second factor auth:

```suggestion
Currently, SMS isn't available for first-factor authentication or self-service password reset in external tenants. However, SMS is now available for second-factor verification in external tenants at additional cost. [Learn more](concept-multifactor-authentication-customers.md#sms-based-authentication)
```